<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotControlWidget</class>
 <widget class="QWidget" name="BotControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>254</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="StartButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PauseButton">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopButton">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
